setup.py

```diff
@@ -26,6 +26,7 @@
 
 EXTRAS_REQUIRE = {'aws': ['boto3'], 'etcd': ['python-etcd'], 'etcd3': ['python-etcd'],
                   'consul': ['py-consul'], 'exhibitor': ['kazoo'], 'zookeeper': ['kazoo'],
+                  'systemd': ['systemd-python'],
```

Comment:
please also add it to requirements.txt, maybe with specifying a minimal version.

Reply:
I did not add a minimal version, since the feature has been available since the first versions of the package.